examples/example.hpp

```diff
@@ -1072,7 +1072,7 @@ void register_glfw_callbacks(window& app, glfw_state& app_state)
 }
 
 // Find devices with specified streams
-std::string depth_with_stream_type_present(rs2_stream type)
+void device_with_streams(rs2_stream type, std::string& out_serial)
 {
     rs2::context ctx;
     for (auto dev : ctx.query_devices(RS2_PRODUCT_LINE_DEPTH))
```

Comment:
remove `RS2_PRODUCT_LINE_DEPTH` filter as it will block both RS2_PRODUCT_LINE_TRACKING and Webcams